shared/common-adapters/popup-menu.desktop.js

```diff
@@ -11,24 +11,47 @@ class Menu extends Component<void, Props, void> {
     }
 
     const realCSS = `
-    .menu-hover:hover {
-      background-color: ${(this.props.style && this.props.style.hoverColor) || globalColors.blue4}
+    .menu-hover:hover { background-color: ${(this.props.style && this.props.style.hoverColor) || globalColors.blue4}; }
+    .menu-hover-danger:hover { background-color: ${globalColors.red}; }
+
+    .menu-hover .title { color: ${globalColors.black_75}; }
+    .menu-hover-danger .title { color: ${globalColors.red}; }
+    .menu-hover-danger:hover .title { color: ${globalColors.white}; }
+    .menu-hover-danger .subtitle { color: ${globalColors.black_40}; }
+    .menu-hover-danger:hover .subtitle { color: ${globalColors.white}; }
     }
     `
 
     return (
-      <Box style={{...stylesMenuCatcher, ...this.props.style}} onClick={() => this.props.onHidden()}>
+      <Box style={{...stylesMenuCatcher}} onClick={e => {
+        this.props.onHidden()
+        e.stopPropagation()
+      }}>
         <style>{realCSS}</style>
-        <Box style={stylesMenu}>
-          {this.props.items.map(i => (
-            <Text key={i.title} className='menu-hover' type='Body' style={{...stylesMenuText, ...i.style}} onClick={i.onClick}>{i.title}</Text>
-          ))}
+        <Box style={{...stylesMenu, ...this.props.style}}>
+          <Box style={{...globalStyles.flexBoxColumn, flexShrink: 0, paddingTop: 7, paddingBottom: 7}} >
+            {this.props.items.map((i, idx) => (i ? (
```

Comment:
I think the structure of the `?` operator here is a bit hard to read since you have to hunt for the `:` at the end. Perhaps assign `separator = <Box ...>` and flip the order so it reads: `!i ? separator : ...` so the logic clearer?